### PR TITLE
updated Kraken to version 2.0.8_beta

### PR DESCRIPTION
**Pull request by Takadonet**
:information_source:
If this is your first pull request to Bioconda please ping `@bioconda/core` so it can be reviewed and to request being added as a member of the Bioconda team. Members of the Bioconda team are able to merge their own pull requests once tests and linting have passed.

----------------

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.

### CODE REVIEW

**Comment by druvus** [comment]
@Takadonet Please also see #14971

**Comment by Takadonet** [comment]
Well then... normally I check to see if someone else was working on it but not this time. Got burned for it this time! 

Will close this MR

**Comment by druvus** [comment]
@Takadonet the update is failing for osx so your help is appreciated. I just wanted to coordinate the efforts to fix it.

**Comment by Takadonet** [comment]
I saw that. Will take a look on my end for the osx issue.

File: recipes/kraken2/meta.yaml [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 source:
-  url: https://github.com/DerrickWood/kraken2/archive/v2.0.7-beta.tar.gz
+  url: https://github.com/DerrickWood/kraken2/archive/v2.0.8-beta.tar.gz
   sha256: {{ sha256 }}
   patches:
     - Makefile.patch
 
 build:
-  number: 3
+  number: 1
```
Comment (dfornika) [inline recipes/kraken2/meta.yaml]: Should the build number be set to `0`?
https://bioconda.github.io/guidelines.html#build-numbers

Comment (Takadonet) [reply recipes/kraken2/meta.yaml]: Typo!

**Comment by dfornika** [comment]
This is looking good to me. I'm eager to see this merged so I can use it for ongoing development of the kraken2 data manager (https://github.com/galaxyproject/tools-iuc/pull/2340).

**Pull request merged**